### DISCUSSION TITLE
Please never require the "For you" tab

### DISCUSSION THREAD

**Discussion by toastal** [community/community#18845] [Other Feature Feedback, Questions, & Ideas] [answered]
This information in this tab is largely garbage. I don't need or want a social network. Unlike the original feed which had selectors that made it pretty easy to hide content that is noisy (like every starred project) and is limited only to people I follow, it's manageable. The "For you" feed is trying to force 'engagement' showing all sorts of stuff from people I don't follow and it doesn't contain useful ARIA or CSS classes to hide nor does it explain why I'm being shown content from projects I don't follow. I see there is the ability to filter some of it, but it's not as refined like I'd like. Instead what will happen is I'll start unstarring everything to remove the noise.

For the time being, I'll use uBlock Origin to hide the tab, but I have a bad feeling that this is supposed to replace the current feed in due time.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by toastal** [reply]
Yep. I saw this a while back which helps. Ultimately this *is* a social media platform so there's not much I can do.
